snowdrop-util/src/Snowdrop/Util/Lens.hs

```diff
@@ -28,7 +28,7 @@ class HasGetter s a => HasLens s a where
 instance HasGetter (NonEmpty a) a where
     gett (x :| _) = x
 
-instance HasGetter a a where
+instance {-# OVERLAPPABLE #-} HasGetter a a where
```

Comment:
I remember the time when you were against `{-# OVERLAPPABLE #-}` syntax

Reply:
I'm still not a fan of it. This case I decided to apply this as quick fix to resolve the reason for the `{-# OVERLAPPABLE #-}` later